server/api.js

```diff
@@ -134,4 +135,64 @@ router.get("/tenders", async (req, res) => {
 	}
 });
 
+router.get("/tender/:id", async (req, res) => {
+	const tenderID = parseInt(req.params.id);
+	let page = parseInt(req.query.page) || 1;
+	const itemsPerPage = 10;
+	const totalBids = await db.query(
+		"SELECT COUNT(tender_id) FROM bid WHERE tender_id = $1",
+		[tenderID]
+	);
+	const totalPages = Math.ceil(totalBids.rows[0].count / itemsPerPage);
+	const offset = (page - 1) * itemsPerPage;
+
+	const result = await db.query(
+		"SELECT * FROM bid WHERE tender_id = $1 LIMIT $2 OFFSET $3",
```

Comment:
I'm confused. Are you retrieving tenders or bids? If you are retrieving bids, then the endpoint should be `/bid` and then you can have a `tender_id` query parameter if you want to filter by tender

Reply:
Yes! You are right. I have fixed it now.

Reply:
Shall I remove the tender_id?